### PR TITLE
feat(hrui): UI integration

### PR DESCRIPTION
**Pull request by dr1rrb**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by carldebilly — APPROVED** [review]
LGTMAOM

File: src/Uno.UI.RemoteControl/HotReload/HotReloadStatusView.Entries.cs [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+			}
+		}
+
+		var remaining = total - count;
+		if (remaining > 0)
+		{
+			sb.Append((count, remaining) switch
+			{
+				(0, 1) => $"1 {kind}",
+				(0, _) => $"{remaining} {kind}s",
```
Comment (carldebilly) [inline src/Uno.UI.RemoteControl/HotReload/HotReloadStatusView.Entries.cs:202]: No all words can be pluralized that way ;-)

Comment (dr1rrb) [reply src/Uno.UI.RemoteControl/HotReload/HotReloadStatusView.Entries.cs:202]: No, but the 2 that we are using are ;)

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.3

**Pull request merged**